.github/workflows/schema.yml

```diff
@@ -0,0 +1,32 @@
+name: Update JSON schema 
+
+on:
+  push:
+		paths:
+		- ../../schemas/'schema.json'
+
+jobs:
+  update-schema:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - uses: actions/checkout@v2
+        with:
+          repository: jina-ai/api
+          path: schema
+          token: ${{ secrets.JINA_DEV_BOT }}
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - uses: actions/setup-node@v2
```

Comment:
You can remove this. We don't need node here.

Reply:
Sure